### PR TITLE
Changed curie to unique id for disease annotations

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/dao/GeneDiseaseAnnotationDAO.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ protected GeneDiseaseAnnotationDAO() {
     }
 
     public List<String> findAllAnnotationCuries(String taxonID) {
```
Comment (markquintontulloch) [inline src/main/java/org/alliancegenome/curation_api/dao/GeneDiseaseAnnotationDAO.java:18]: Also rename?

File: src/main/java/org/alliancegenome/curation_api/dao/AlleleDiseaseAnnotationDAO.java [hunk 2]
```diff
@@ -16,7 +16,7 @@ protected AlleleDiseaseAnnotationDAO() {
     }
 
     public List<String> findAllAnnotationCuries(String taxonID) {
```
Comment (markquintontulloch) [inline src/main/java/org/alliancegenome/curation_api/dao/AlleleDiseaseAnnotationDAO.java:18]: Also rename?

File: src/main/java/org/alliancegenome/curation_api/dao/AGMDiseaseAnnotationDAO.java [hunk 3]
```diff
@@ -16,7 +16,7 @@ protected AGMDiseaseAnnotationDAO() {
     }
 
     public List<String> findAllAnnotationCuries(String taxonID) {
```
Comment (markquintontulloch) [inline src/main/java/org/alliancegenome/curation_api/dao/AGMDiseaseAnnotationDAO.java:18]: Maybe we should change the method name to findAllAnnotationUniqueIds or something similar?

File: src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java [hunk 4]
```diff
@@ -177,7 +177,7 @@ public void runLoad(String taxonID, DiseaseAnnotationMetaDataDTO annotationData)
         annotationData.getData().forEach(annotationDTO -> {
             DiseaseAnnotation annotation = upsert(annotationDTO);
             if (annotation != null) {
-                annotationsCuriesAfter.add(annotation.getCurie());
+                annotationsCuriesAfter.add(annotation.getUniqueId());
```
Comment (markquintontulloch) [inline src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java:180]: Rename annotationsCuriesAfter?

**Review by markquintontulloch — APPROVED** [review]
Logic looks good - I wonder if it's worth renaming the mentioned methods and variables to avoid confusion.  I'm not too bothered though.

File: src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java [hunk 5]
```diff
@@ -192,7 +192,7 @@ public void runLoad(String taxonID, DiseaseAnnotationMetaDataDTO annotationData)
         log.debug("runLoad: Remove: " + taxonID + " " + curiesToRemove.size());
         
         for (String curie : curiesToRemove) {
```
[the remark this answers is not in the document]
Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java:194]: Yes you are right...

**Pull request merged**